tensorboard/components/tf_globals/globals-polymer.ts

```diff
@@ -20,4 +20,11 @@ import * as tf_globals from './globals';
 class TfGlobals extends PolymerElement {
   override _template = null;
   tf_globals = tf_globals;
+  constructor() {
+    super();
+    if (!window.tensorboard) {
+      window.tensorboard = {};
+    }
+    window.tensorboard.tf_globals = tf_globals;
```

Comment:
Note that the change doesn't import cleanly:

third_party/tensorboard/components/tf_globals/globals-polymer.ts:25:17 - error TS2339: Property 'tensorboard' does not exist on type 'Window & typeof globalThis'.

25     if (!window.tensorboard) {
                   ~~~~~~~~~~~
third_party/tensorboard/components/tf_globals/globals-polymer.ts:26:14 - error TS2339: Property 'tensorboard' does not exist on type 'Window & typeof globalThis'.

26       window.tensorboard = {};
                ~~~~~~~~~~~
third_party/tensorboard/components/tf_globals/globals-polymer.ts:28:12 - error TS2339: Property 'tensorboard' does not exist on type 'Window & typeof globalThis'.

28     window.tensorboard.tf_globals = tf_globals;
              ~~~~~~~~~~~